CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 7.6.0 (2023-02-08)
```

Comment:
These are all bug fixes - think this should be a patch release.

Reply:
> Actually, @rich-bugsnag, I have just noticed that GitHub is saying that the branch is out of date and to merge `master` into `next`. Is that normal? I suspect it might be a good idea to do what it says at this stage of proceedings.

All good, done 👍

Reply:
> These are all bug fixes - think this should be a patch release.

Changed to v7.5.1 👍